widget/scroller.go

```diff
@@ -354,6 +354,24 @@ func (s *ScrollContainer) CreateRenderer() fyne.WidgetRenderer {
 	return scr
 }
 
+
+//ScrollToBottom will scroll content to container bottom - to show latest info which end user just added
+func (s *ScrollContainer) ScrollToBottom(isNeedRefresh bool) {
```

Comment:
I didn't notice the isNeedRefresh flag get added - it feels unexpected.
Normally widgets refresh after any change function, is there a problem with doing that here?

Reply:
In one of my scenario, the refresh is needed. So, better let end user to select must refresh or not.

Reply:
But if you set this value and don't call refresh then it will not update the display - how can it be valid to not refresh?

Reply:
Remove that isNeedRefresh flag.